gcovr/tests/shadow/Makefile

```
@@ -0,0 +1,29 @@
+all:
+	cd build/ && $(MAKE) -f Makefile all
```

There is no Makefile in build, so:

```suggestion
	cd build/ && $(MAKE) -f ../Makefile all
```

---

Forget is :)